<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Hub</class>
 <widget class="QMainWindow" name="Hub">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hub</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTextBrowser" name="text_browser"/>
    </item>
    <item row="1" column="0">
     <widget class="QTextBrowser" name="topic_browser"/>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="start_sim_button">
      <property name="text">
       <string>start sim</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
